pandas/tseries/base.py

```diff
@@ -26,7 +25,7 @@ def strftime(self, date_format):
         Return an array of formatted strings specified by date_format, which
         supports the same string format as the python standard library. Details
         of the string format can be found in the `python string format doc
-        <https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior>`__
+        https://docs.python.org/2/library/datetime.html.
```

Comment:
This is hard, because there's no easy way (AFAICT, manual string concatenation is all I could think of) to keep the RST valid while also suppressing the flake8 warning. I fixed the broken markup for now. I've always felt that docstrings should be optimized for console use, and that more complex documentation should be in the docs themselves, but it's not a big deal either way